modules/mobile/app/controllers/mobile/v0/payment_information_controller.rb

```diff
@@ -32,15 +32,26 @@ def index
       end
 
       def update
+        lh_error_response = nil
+
         payment_information = if lighthouse?
-                                lighthouse_adapter.parse(lighthouse_service.update_payment_info(pay_info))
+                                begin
+                                  lighthouse_adapter.parse(lighthouse_service.update_payment_info(pay_info))
+                                rescue => e
```

Comment:
I think you should consider only rescuing sub-classes of `Common::Exceptions::BaseError`. Not all errors will have a `status_code` method, so the next line will fail if, for example, we encounter a ruby nil error.